go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/vbatts/tar-split
+
+go 1.15
```

Comment:
This could be set as low as 1.11 (IIRC), and that might help the Go tool understand the level of compatibility you're shooting for? :innocent:

Reply:
Fair. The travis test went back to g1.10 and that's green, sooo...